Tests/ProjectDescriptionTests/EnvironmentTests.swift

```diff
@@ -63,4 +63,35 @@ final class EnvironmentTests: XCTestCase {
         let value = Environment.value(for: "1", environment: environment)
         XCTAssertNil(value)
     }
+
+    func testValueChecksForTuistPrefixedValuesFirst() {
+        let environment: [String: String] = [
+            "TUIST_NAME_SUFFIX": "0",
+            "NAME_SUFFIX": "1",
+        ]
+
+        // mimicing the camel cased dynamic member format
```

Comment:
```suggestion
        // mimicking the camel cased dynamic member format
```

Reply:
This is a good catch, but on an outdated commit. I had already pushed this change after it looked off to me in the diff 😄